### PR TITLE
Send a warning instead throwing an exeption when message type is not …

### PR DESCRIPTION
**Pull request by fgallaire**
…compatible with as_copy
As messages copies are often done in batches, send a warning and continue the work is a far better behavior (so much user friendly).

### CODE REVIEW

**Comment by fgallaire** [comment]
@delivrance do you prefer a `no_fail` boolean for retro-compatibility reason ?

**Comment by delivrance** [comment]
I tried to forward three messages, two texts and one service in the middle; the server successfully forwards the two texts. I approve this.

**Comment by delivrance** [comment]
> @delivrance do you prefer a `no_fail` boolean for retro-compatibility reason ?

I'd rather keep the code simpler and mention in the release note about this possible breaking change. Thanks

File: pyrogram/client/types/messages_and_media/message.py [hunk 1]
```diff
@@ -2684,12 +2687,10 @@ def forward(
         """
         if as_copy:
             if self.service:
-                raise ValueError("Unable to copy service messages")
-
-            if self.game and not self._client.is_bot:
-                raise ValueError("Users cannot send messages with Game media type")
-
-            if self.text:
+                log.warning("Unable to copy service messages, message_id: {}".format(self.message_id))
```
Comment (delivrance) [inline pyrogram/client/types/messages_and_media/message.py:2690]: Maybe also mention the source chat id (`self.chat.id`) and not only the message id.

**Pull request merged**